### ISSUE TITLE
Problem completing the Basic Usage with ROS

### ISSUE THREAD

**Issue by sotomotocross** [uzh-rpg/flightmare#26]
I have successfully made catkin build my workspace (only one workspace for the flightmare simulator) and i try to complete the Basic Usage with ROS.
I downloaded RPG_Flightmare, i extracted it into the /path/to/flightmare/flightrender and run roslaunch flightros rotors_gazebo.launch as indicated.
Therefore, i have and Error in REST request and also the Unity Connection time out! 
![Screenshot from 2020-09-23 16-00-01](https://user-images.githubusercontent.com/23704278/94016181-7b232f00-fdb6-11ea-9219-7f158ba253d0.png)
Is there a proper way of extracting the contents of RPG_Flightmare? Should i change something or the extracted just has to be inside the flightrender directory?
Is there anothe problem?
Also if i run: roslaunch flightros rotors_gazebo_test.launch
then i have the error below:
![Screenshot from 2020-09-23 16-06-16](https://user-images.githubusercontent.com/23704278/94016419-c1788e00-fdb6-11ea-901f-d950fd71fca2.png)
Looking forward for any help.
Thank you very much.

**Comment by sotomotocross** [comment]
Quick correction i changed the name of RPG_Flightmare.x86_64
The roslaunch flightros rotors_gazebo_test.launch runs ok.
Although if i run roslaunch flightros rotors_gazebo.launch all the errors indicated at the beginning remain.

**Comment by yun-long** [comment]
can you first start the Unity Standalone by double-clicking the RPG_Flightmare.x86_64, and then, roslaunch flightros rotors_gazebo.launch?

**Comment by sotomotocross** [comment]
Yes i double-clicked on the RPG_Flightmare.x86_64 and then roslaunch flightros rotors_gazebo.launch and everything started ok.
I hadn't understood that i have to run it like this before beginning the ROS.
Thank you very much.

**Comment by yun-long** [comment]
It is implemented in the roslaunch file, https://github.com/uzh-rpg/flightmare/blob/master/flightros/launch/rotors_gazebo.launch#L68-L69

You can change the default value for `use_unity_editor`  to false (Which means you don't run unity editor but run the unity standalone called RPG_Flightmare.x86_64). The ros will launch RPG_Flightmare.x86_64 automatically.

**Comment by sotomotocross** [comment]
I changed the value of use_unity_editor to false and the  roslaunch flightros rotors_gazebo.launch worked very good.
Thank you very much for the help.

**Comment by Reka-Berci-Hajnovics** [comment]
I have run into the same issue. @yun-long  Could you please consider adding a Troubleshooting section in the _Basic usage with ROS_ page?
I think it would be a helpful tip for other users as well.

**Comment by vivekrathi14** [comment]
Hi,
I am having similar issue;

System Specifications -
VMWare Ubuntu 18.04, ROS Melodic

I followed all instructions from wiki and was able to install and build the packages in the workspace.
Further I downloaded the RPG_Flightmare and extracted it into `<workspace>/src/flightmare/filghtrender`
I tried running` roslaunch rotors_gazebo_test.launch`

It gives me this **error**:
`[rpg_flightmare_render-3] process has died`

I tried to run the executable but I get this error:
![image](https://user-images.githubusercontent.com/25856691/98262477-a96a6180-1f53-11eb-81e6-168702121e0b.png)

However, if I use sudo ./RPG_Flightmare.x86_64 it works.

Can someone help me here to get ROS example working?
